site/content/how-to/monitoring/prometheus.md

```diff
@@ -11,13 +11,91 @@ docs: "DOCS-1418"
 
 {{<call-out "important" "Security note for metrics">}}
 Metrics are served over HTTP by default. Enabling HTTPS will secure the metrics endpoint with a self-signed certificate. When using HTTPS, adjust the Prometheus Pod scrape settings by adding the `insecure_skip_verify` flag to handle the self-signed certificate. For further details, refer to the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#tls_config).
 {{</call-out>}}
 
+## Installing Prometheus and Grafana
+
+### Prometheus
+
+```shell
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install prometheus prometheus-community/prometheus -n monitoring --create-namespace --set server.global.scrape_interval=15s
+```
+
+Once running, you can access the Prometheus dashboard by using port-forwarding in the background:
+
+```shell
+kubectl port-forward -n monitoring svc/prometheus-server 9090:80 &
+```
+
+Visit [http://127.0.0.1:9090](http://127.0.0.1:9090) to view the dashboard.
+
+### Grafana
+
+
+```shell
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+helm install grafana grafana/grafana -n monitoring --create-namespace
+```
+
+Once running, you can access the Grafana dashboard by using port-forwarding in the background:
+
+```shell
+kubectl port-forward -n monitoring svc/grafana 3000:80 &
+```
+
+Visit [http://127.0.0.1:3000](http://127.0.0.1:3000) to view the Grafana UI.
+
+The username for login is `admin`. The password can be acquired by running:
+
+```shell
+kubectl get secret -n monitoring grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+```
+
+#### Configuring Grafana
+
+In the Grafana UI menu, go to `Connections` then `Data sources`. Add your Prometheus service (`http://prometheus-server.monitoring.svc`) as a data source.
+
+Download the following sample dashboard and Import as a new Dashboard in the Grafana UI.
+
+{{< fa "download" >}} {{< link "grafana-dashboard.json" "`ngf-grafana-dashboard.json`" >}}
+
+## Available metrics in NGINX Gateway Fabric
+
+NGINX Gateway Fabric provides a variety of metrics to assist in monitoring and analyzing performance. These metrics are categorized as follows:
+
+### NGINX/NGINX Plus metrics
+
+NGINX metrics, essential for monitoring specific NGINX operations, include details like the total number of accepted client connections. For a complete list of available NGINX/NGINX Plus metrics, refer to the [NGINX Prometheus Exporter developer docs](https://github.com/nginxinc/nginx-prometheus-exporter#exported-metrics).
+
+These metrics use  the `nginx_gateway_fabric` namespace and include the `class` label, indicating the NGINX Gateway class. For example, `nginx_gateway_fabric_connections_accepted{class="nginx"}`.
+
+### NGINX Gateway Fabric metrics
+
+Metrics specific to the NGINX Gateway Fabric include:
```

Comment:
```suggestion
Metrics specific to NGINX Gateway Fabric include:
```

Any usage of "the" prepending software names suggests a specific instance, whereas this is describing it in a generic sense.